public/js/components/FormFields/FormFieldScribeEditor.js

```diff
@@ -109,14 +117,16 @@ export class ScribeEditor extends React.Component {
       }
     }));
 
+    this.scribe.on('content-changed', this.onContentChange);
+    this.state.scribeElement.innerHTML = this.props.value; // eslint-disable-line react/no-direct-mutation-state
```

Comment:
You can avoid switching off eslint here by doing something along the lines of: 
```js
// Creates a copy of the scribeElement with the new innerHtml
const newElement = Object.assign({}, this.state.scribeElement, {
  innerHtml: this.props.value
}

// Changes the scribeElement in state
this.setState({
  scribeElement: newElement
});
```

It's less destructive and more reacty too